.github/workflows/release.yml

```diff
@@ -17,6 +17,8 @@ jobs:
           ruby-version: '3.3'
         env:
           BUNDLE_WITHOUT: release
+      - name: Generate database
+        run: bundle exec rake database
```

Comment:
should we update it on each release or on each merge?

Reply:
mhm what does that actually do? just build a yard db that will be added to the gem before it's uploaded?

Reply:
It builds the old table from the README and several lists sorted in different ways. @ekohl had added it to the `.gitignore` in PR #262, so I kept it that way, meaning this would be run on release. Does it make sense to run on every merge instead and remove from the `.gitignore`? Run on every merge and just update GH pages?

Reply:
You can see the output from YARD here: https://www.cs.umd.edu/~ykatz/facterdb/file.list_arch_os_facter.html

Reply:
I'm a big fan of not including generated files in git, so will generally approve PRs that avoid it. That said, if this job doesn't show it then I assume it's verify the job passes. Is that correct?